calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -52,6 +52,16 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 - WAF is not supported for host-networked client pods
 - When selecting and deselecting traffic for WAF, active connections may be disrupted
 
+:::note warning
```

Comment:
```suggestion
:::caution
```
See https://docusaurus.io/docs/markdown-features/admonitions. 

You've created something that renders, but it's not quite what you were going for, I think. Is this right?

Reply:
You're right. Fixed it.